runtime/parser/declaration.go

```diff
@@ -214,7 +214,7 @@ func parseDeclaration(p *parser, docString string) (ast.Declaration, error) {
 				purity = parsePurityAnnotation(p)
 				continue
 
-			case KeywordPriv, KeywordPub, KeywordAccess:
```

Comment:
Wondering if we should parse, but report syntax error. This would help developers when migrating